dawn/src/dawn/IIR/ASTStmt.h

```diff
@@ -53,9 +72,24 @@ std::shared_ptr<ast::ReturnStmt> makeReturnStmt(Args&&... args) {
   return std::make_shared<ast::ReturnStmt>(std::make_unique<IIRStmtData>(),
                                            std::forward<Args>(args)...);
 }
+
+struct VarDeclStmtData : public IIRStmtData {
+
+  VarDeclStmtData() = default;
+  VarDeclStmtData(const VarDeclStmtData&);
+
+  bool operator==(const VarDeclStmtData&);
+  bool operator!=(const VarDeclStmtData&);
+
+  /// ID of the variable declared by the statement
+  std::optional<int> AccessID;
```

Comment:
will write `iir::getAccessIDFromStmt()` ... I checked the naming conventions for other public fields of classes/structs and `AccessID` is correct (see structs in `SIR/SIR.h`)

Reply:
Or those are also wrong, because the naming convention that was shared initially was different: https://meteoswiss-apn.github.io/dawn/coding.html#style-issues

Reply:
I am not sure if that's still up to date... It seems to me that there's this (unwritten) convention of having public members of classes starting with a capital letter ... for consistency I would keep `AccessID` until a decision is taken for all

Reply:
okay. Let's try to remember bringing it up :)